client/src/components/BubblePage.js

```diff
@@ -3,17 +3,29 @@ import axios from "axios";
 
```

Comment:
make commits more often so your thought process and work can be seen, also you want to be able to go back to a working commit, when the projects get larger